### PR TITLE
Add support for the new categorize_and_describe endpoint

### PR DESCRIPTION
**Pull request by brettmortensen**
@film42 @newellista

### CODE REVIEW

File: src/main/java/atrium/AtriumClient.java [hunk 1]
```diff
@@ -577,6 +577,28 @@ public Account readAccount(String userGUID, String accountGUID) {
 
   // TRANSACTION
 
+  /**
+   * Method used to categorize and describe a Transaction.
+   * @param transactions Required. An array of RawTransaction objects.
+   * @return CategorizedTransaction[]. This returns an array of CategorizedTransaction objects.
+   */
+  public CategorizedTransaction[] categorizeAndDescribeTransactions(JsonArray transactions) {
```
Comment (liveh2o) [inline src/main/java/atrium/AtriumClient.java]: What's the thinking behind returning a different object type here? Is there something that the `CategorizedTransaction` provides that the `Transaction` does not?

Comment (film42) [inline src/main/java/atrium/AtriumClient.java]: Why a JsonArray and not something like `List<Transaction> transactions`?

Comment (brettmortensen) [reply src/main/java/atrium/AtriumClient.java]: Thanks for pointing that out. I hadn't realized it was a subset. Removed `CategorizedTransaction` in favor of `Transaction`.

Comment (brettmortensen) [reply src/main/java/atrium/AtriumClient.java]: Updated. This approach is the way to go. 👍 

I originally went with `JsonArray` as it was the convention we were using in a few other endpoints in this library. I'll be getting up another PR soon to update the other endpoints. (https://github.com/mxenabled/atrium-java/pull/5)

File: src/main/java/atrium/AtriumClient.java [hunk 2]
```diff
@@ -577,6 +578,29 @@ public Account readAccount(String userGUID, String accountGUID) {
 
   // TRANSACTION
 
+  /**
+   * Method used to categorize and describe a Transaction.
+   * @param transactions Required. An array of RawTransaction objects.
+   * @return Transaction[]. This returns an array of Transaction objects.
+   */
+  public Transaction[] categorizeAndDescribeTransactions(List<Transaction> transactions) {
```
Comment (film42) [inline src/main/java/atrium/AtriumClient.java:586]: Response should probably be a List type.

**Review by film42 — CHANGES_REQUESTED** [review]

**Comment by brettmortensen** [comment]
This PR is no longer necessary with https://github.com/mxenabled/atrium-java/pull/7

**Pull request closed without merging**